apps/sap-commerce-cloud/frontend/package.json

```diff
@@ -17,7 +17,8 @@
     "cross-env": "^7.0.3",
     "lodash.get": "^4.4.2",
     "react": "^18.3.1",
-    "react-dom": "^18.3.1"
+    "react-dom": "^18.3.1",
+    "use-debounce": "^10.0.3"
```

Comment:
Popular library been around a while, saw a similar thing we were using for other apps - a bit more straightforward to implement than the lodash one with a callback.

Reply:
we could also just implement our own debounce function:

```
function debounce(func, delay) {
  let timeoutId;

  return function(...args) {
    clearTimeout(timeoutId);

    timeoutId = setTimeout(() => {
      func.apply(this, args);
    }, delay);
  };
}
```